app/views/outcomes-report/ClanLeagueStatsComponent.vue

```diff
@@ -0,0 +1,158 @@
+<script>
+import utils from 'core/utils'
+export default Vue.extend({
+  name: 'AILeagueStatsComponent',
+  props: {
+    stats: {
+      type: Object,
+      default () {
+        return {}
+      }
+    },
+    myId: {
```

Comment:
I can't see in the code where is this prop used?

Reply:
oh thank you. i remove all usage and now forget to clean it